mq/main.tf

```diff
@@ -0,0 +1,41 @@
+locals {
+  common_tags = {
+    "Name" = var.broker_name
+    "Purpose" = var.broker_purpose
+    "Owner" = var.broker_owner
+  }
+}
+resource "aws_mq_broker" "mq" {
```

Comment:
I don't see an option to create in specific VPC or subnet. Excluding it , will just create it in default vpc and subnet. Not sure if we have to consider that use case for now. Just thinking in general perspective if we want to use these scripts somewhere else.